src/main/java/org/tailormap/api/persistence/GeoService.java

```diff
@@ -95,6 +96,7 @@ public class GeoService {
    * Original capabilities as received from the service. This can be used for capability information not already
    * parsed in this entity, such as tiling information.
    */
+  @NotAudited
```

Comment:
see https://discourse.hibernate.org/t/envers-fails-when-a-field-is-edited-and-another-field-is-lazy-and-not-loaded/9842